src/StripeTests/StripeTests.csproj

```diff
@@ -6,6 +6,7 @@
     <AssemblyName>StripeTests</AssemblyName>
     <PackageId>StripeTests</PackageId>
     <RuntimeIdentifiers>win10-x64</RuntimeIdentifiers>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
```

Comment:
It's surprising to enable documentation output in a test project, but apparently disabling this can cause strange issues with the compiler, so it's recommended to enable it even if not needed: https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/2192#issuecomment-241396037